setup.py

```diff
@@ -21,14 +21,11 @@
     "autorom[accept-rom-license]~=0.4.2",
 ]
 PYTYPE = ["pytype==2022.7.26"] if IS_NOT_WINDOWS else []
-if IS_NOT_WINDOWS:
-    # TODO(adam): use this for Windows as well once PyPI is at >=1.6.1
-    STABLE_BASELINES3 = "stable-baselines3>=1.6.0"
-else:
-    STABLE_BASELINES3 = (
-        "stable-baselines3@git+"
-        "https://github.com/DLR-RM/stable-baselines3.git@master"
-    )
+STABLE_BASELINES3 = "stable-baselines3>=1.6.1"
+# STABLE_BASELINES3 = (
```

Comment:
Delete this?

As an aside, I see commented out code in your PRs not that infrequently. Consider doing a quick self-review or adding a pre-commit warning for things like this that come up repeatedly? Alternatively, when I'm making small changes I often like to stage changes with `git add -p` so I'm scrutinizing everything I'm adding to the branch (but still have the freedom to make quick and dirty changes during development that I leave unstaged and clean up later).